package/src/contexts/threadContext/ThreadContext.tsx

```diff
@@ -54,17 +54,22 @@ export const useThreadContext = <
 
   return contextValue;
 };
+
 /**
- * Typescript currently does not support partial inference so if ThreadContext
- * typing is desired while using the HOC withThreadContextContext the Props for the
+ * @deprecated because of no usage
```

Comment:
can we remove `because of no usage` in all equivalent comments please.. 

the reason below is acceptable enough in my view